testng-core/src/test/kotlin/org/testng/reporters/jq/TimesPanelTest.kt

```diff
@@ -0,0 +1,29 @@
+package org.testng.reporters.jq
+
+import org.assertj.core.api.Assertions.assertThat
+
+import org.testng.annotations.Test
+import org.testng.internal.paramhandler.FakeSuite
+import org.testng.reporters.XMLStringBuffer
+import test.SimpleBaseTest
+
+class TimesPanelTest: SimpleBaseTest() {
```

Comment:
Don't inherite from SimpleBaseTest.

Reply:
We are using some helpers from `SimpleBaseTest`. So we need to be extending it